projects/fal/src/fal/toolkit/file/providers/fal.py

```diff
@@ -70,3 +72,40 @@ def _upload_file(self, upload_url: str, file: FileData):
 class InMemoryRepository(FileRepository):
     def save(self, file: FileData) -> str:
         return f'data:{file.content_type};base64,{b64encode(file.data).decode("utf-8")}'
+
+
+@mainify
+@dataclass
+class FalCDNFileRepository(FileRepository):
+    def save(self, file: FileData) -> str:
+        headers = {
+            **self.auth_headers,
+            "Accept": "application/json",
+            "Content-Type": file.content_type,
+        }
+
+        url = _FAL_CDN + "/files/upload"
```

Comment:
@badayvedat can we do `os.getenv("FAL_CDN_HOST", _FAL_CDN)`  so when we change the URL it will take it effect for everyone who has the code deployed?